tgui/packages/tgui/interfaces/OutfitEditor.js

```diff
@@ -0,0 +1,138 @@
+import { useBackend } from '../backend';
+import { Button, Section, Box, Stack } from '../components';
+import { Window } from '../layouts';
+
+export const OutfitEditor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { outfit, saveable } = data;
```

Comment:
dummy64 should be in this data, in a few spots i see you using data.thing but that's bad for readability